lib/internal/modules/cjs/loader.js

```diff
@@ -134,30 +134,34 @@ const wrapper = [
 ];
 
 let wrapperProxy = new Proxy(wrapper, {
-  set(obj, prop, value) {
-    if (prop === '0' || prop === '1') {
-      return Reflect.set(...arguments);
-    }
+  set(obj, prop, value, receiver) {
+    patched = true;
+    return Reflect.set(...arguments);
+  },
+
+  defineProperty(target, property, descriptor) {
+    patched = true;
+    return Object.defineProperty(...arguments);
```

Comment:
please don't use `arguments`... all the arguments are right there.

Reply:
Okay.